2021/25xxx/CVE-2021-25740.json

```
@@ -1,18 +1,99 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "security@kubernetes.io",
+        "DATE_PUBLIC": "2021-07-14T21:23:00.000Z",
         "ID": "CVE-2021-25740",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "Holes in EndpointSlice Validation Enable Host Network Hijack"
     },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Kubernetes",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": ">=",
+                                            "version_value": "0.0.0"
+                                        }
+                                    ]
```

This would need to be updated if we ever fix it (e.g. endpoints v2). I'm worried it's a little misleading. Maybe this would be a better option?

```suggestion
                                    "version_data": [
                                        {
                                            "version_affected": "<=",
                                            "version_value": "1.22.2"  # Maybe explicitly enumerate other supported minor versions?
                                        }, 
                                        {
                                            "version_affected": ">?",  # found this option on vulnogram...
                                            "version_value": "1.22.2"
                                        }
                                    ]
```

---

Done. Added for other minor versions as well